### ISSUE TITLE
Math_Vista result is not summarized

### ISSUE THREAD

**Issue by ruifengma** [open-compass/VLMEvalKit#253]
After running the evaluation process on MathVista. There is only a .xlsx file which contains the answer and predictions from model but there is not a finalized result (e.g. accuracy)

**Comment by junming-yang** [comment]
MathVista, MMVet and LLaVABench need to set your API Key to evaluation. You can refer to [QuickStart](https://github.com/open-compass/VLMEvalKit/blob/main/docs/en/Quickstart.md) for more details.

**Comment by ruifengma** [comment]
> MathVista, MMVet and LLaVABench need to set your API Key to evaluation. You can refer to [QuickStart](https://github.com/open-compass/VLMEvalKit/blob/main/docs/en/Quickstart.md) for more details.

yes, I actually followed the instruction and a .env file has been created with 
```
OPENAI_API_KEY=empty
OPENAI_API_BASE=http://10.79.13.172:8100/v1/chat/completions
LOCAL_LLM=Qwen1.5-72B-Chat
```
but got
`Evaluation - ERROR - OPENAI_API_KEY is not set properly, will use exact matching for evaluation`

**Comment by junming-yang** [comment]
OPENAI_API_KEY can not be empty. You should start with `sk-`. And check your API Key in the correct path `$VLMEvalKit/.env`.  You can also set these params to `~/.bashrc` and run `openai_key = os.environ.get('OPENAI_API_KEY', None)` to check.

**Comment by ruifengma** [comment]
` assert isinstance(key, str) and key.startswith('sk-')`
OK, got in the source code. Thanks
